libraries/standard/mqtt/src/mqtt_state.c

```diff
@@ -19,6 +19,10 @@
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
+/**
+ * @file mqtt_state.c
+ * @brief Implements the user-facing functions in mqtt_state.h.
```

Comment:
I don't think all the state functions are user-facing

Reply:
Ah right this needs to be update too thanks